src/pages/EntityDetails/Model/Logs/ActionLogs/ActionLogs.tsx

```diff
@@ -162,14 +161,12 @@ export default function ActionLogs() {
   }>({});
   const queryOperationsList = useQueryOperationsList(userName, modelName);
   const queryActionsList = useQueryActionsList(userName, modelName);
-  const getModelUUIDMemo = useMemo(
-    () => (modelName ? getModelUUID(modelName) : null),
-    [modelName],
+  const modelUUID = useAppSelector((state) =>
+    modelName ? getModelUUID(state, modelName) : null,
```

Comment:
Same here about support a null model name in getModelUUID so you can remove this ternary.